lib/src/time-range-dialog.dart

```diff
@@ -120,39 +120,39 @@ showTimeRangePicker({
   final Widget dialog = Dialog(
       elevation: 12,
       child: TimeRangePicker(
-        start: start,
```

Comment:
Seems that auto formatting from my VS code editor had other opinions compared to what was is this file. I only added the `use24HourFormat` param here to pass it along to the localization function.